src/haswell/bitmanipulation.h

```diff
@@ -58,7 +58,7 @@ really_inline bool add_overflow(uint64_t value1, uint64_t value2,
 #endif
 }
 
-#ifdef SIMDJSON_REGULAR_VISUAL_STUDIO
+#if defined(SIMDJSON_REGULAR_VISUAL_STUDIO) || defined(SIMDJSON_IS_32BITS)
```

Comment:
(Redundant comment.) My expectation is that a 32-bit compiler would never make it there. I don't the code change, but I'd like to check my expectation with you.

Reply:
(Redundant) I manually checked several times on all of these changes. 32-bit GCC (both  "vanilla" Linux, and the MinGW Windows port) make it here.